embark-ui/package.json

```diff
@@ -4,88 +4,89 @@
+    "ethereumjs-units": "0.2.0",
     "file-loader": "2.0.0",
+    "font-awesome": "4.7.0",
     "fork-ts-checker-webpack-plugin-alt": "0.4.14",
     "fs-extra": "7.0.0",
+    "history": "4.7.2",
     "html-webpack-plugin": "4.0.0-alpha.2",
     "identity-obj-proxy": "3.0.0",
     "jest": "23.6.0",
     "jest-pnp-resolver": "1.0.1",
     "jest-resolve": "23.6.0",
+    "keccakjs": "0.2.1",
     "mini-css-extract-plugin": "0.4.3",
+    "monaco-editor": "0.14.3",
+    "monaco-editor-webpack-plugin": "1.6.0",
     "optimize-css-assets-webpack-plugin": "5.0.1",
     "pnp-webpack-plugin": "1.1.0",
     "postcss-flexbugs-fixes": "4.1.0",
     "postcss-loader": "3.0.0",
     "postcss-preset-env": "6.0.6",
     "postcss-safe-parser": "4.0.1",
-    "react": "^16.6.1",
-    "react-app-polyfill": "^0.1.3",
-    "react-dev-utils": "^6.1.1",
-    "react-dom": "^16.6.1",
+    "prop-types": "15.6.2",
+    "qs": "6.5.2",
+    "re-resizable": "4.9.3",
+    "react": "16.6.3",
+    "react-app-polyfill": "0.1.3",
+    "react-blockies": "1.4.0",
+    "react-bootstrap-typeahead": "3.2.4",
+    "react-copy-to-clipboard": "5.0.1",
+    "react-dev-utils": "6.1.1",
+    "react-dom": "16.6.3",
+    "react-fontawesome": "1.6.1",
+    "react-json-view": "1.19.1",
+    "react-redux": "5.1.1",
+    "react-router-dom": "4.3.1",
+    "react-scroll-to-component": "1.0.2",
+    "react-treebeard": "2.1.0",
+    "reactstrap": "6.5.0",
+    "redux": "4.0.1",
+    "redux-saga": "0.16.2",
     "resolve": "1.8.1",
     "sass-loader": "7.1.0",
+    "simple-line-icons": "2.4.1",
     "style-loader": "0.23.0",
     "terser-webpack-plugin": "1.1.0",
     "url-loader": "1.1.1",
+    "web3": "1.0.0-beta.36",
     "webpack": "4.19.1",
     "webpack-dev-server": "3.1.9",
     "webpack-manifest-plugin": "2.0.4",
-    "workbox-webpack-plugin": "3.6.3",
-    "@coreui/coreui": "^2.0.15",
-    "@coreui/react": "^2.0.9",
-    "ansi-to-html": "^0.6.6",
-    "autoscroll-react": "^3.2.0",
-    "axios": "^0.18.0",
-    "classnames": "^2.2.6",
-    "connected-react-router": "^4.3.0",
-    "ethereumjs-units": "0.2.0",
-    "font-awesome": "^4.7.0",
-    "history": "^4.7.2",
-    "keccakjs": "0.2.1",
-    "monaco-editor": "^0.14.3",
-    "monaco-editor-webpack-plugin": "^1.5.4",
-    "prop-types": "^15.6.2",
-    "qs": "^6.5.2",
-    "re-resizable": "^4.9.3",
-    "react-blockies": "^1.4.0",
-    "react-bootstrap-typeahead": "^3.2.4",
-    "react-copy-to-clipboard": "^5.0.1",
-    "react-fontawesome": "^1.6.1",
-    "react-json-view": "^1.19.1",
-    "react-redux": "^5.0.7",
-    "react-router-dom": "^4.3.1",
-    "react-scroll-to-component": "^1.0.2",
-    "react-treebeard": "^2.1.0",
-    "reactstrap": "^6.5.0",
-    "redux": "^4.0.0",
-    "redux-saga": "^0.16.0",
-    "simple-line-icons": "^2.4.1",
-    "web3": "^1.0.0-beta.36"
+    "workbox-webpack-plugin": "3.6.3"
   },
   "scripts": {
-    "start": "node scripts/start.js",
     "build": "node scripts/build.js",
+    "lint": "eslint src/",
```

Comment:
Not 100% sure why (yet) but I had to put the linting of `embark-ui/src` inside an embark-ui npm-script, otherwise there was a dependency resolution problem.

Reply:
even better IMO